src/client/components/Graphs.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react'
+
+interface GraphsState {
+  data: any[]
+}
+
+class Graphs extends Component<{}, GraphsState> {
```

Comment:
Since this is a new component, look into using Hooks instead of classes. To manage state, you would use the `useState` hook: https://reactjs.org/docs/hooks-state.html